src/bundles/unity_academy/UnityAcademy.tsx

```diff
@@ -542,13 +579,23 @@ class UnityAcademyJsInteropContext {
     this.guiData.push(newButton);
   }
 
-  setTargetFrameRate(newTargetFrameRate : number) {
+  setTargetFrameRate(newTargetFrameRate : number) : void {
     newTargetFrameRate = Math.floor(newTargetFrameRate);
     if (newTargetFrameRate < 15) return;
     if (newTargetFrameRate > 120) return;
     this.targetFrameRate = newTargetFrameRate;
   }
 
+  setCustomPropertyInternal(gameObjectIdentifier : GameObjectIdentifier, propName : string, value : any) : void {

```

Comment:
@leeyi45 raised concerns about the propName parameter: he was thinking there was a XSS vulnerability here. I think that's not the case, because the entered string is going to remain uninterpreted as a string at runtime. It cannot escape being a string: The string is not going to be subjected to any language processing (e.g. compilation).